client/core/qb.lua

```diff
@@ -25,43 +24,48 @@ end)
 
 local shared = exports[coreName]:GetCoreObject()
 
-local coreFunctionsOverride = {
-    Functions = {
-        playerData = {
-            originalMethod = 'GetPlayerData',
-            modifier = {
-                executeFun = true,
-                effect = function(originalFun)
-                    while not LocalPlayer.state.isLoggedIn do
-                        Wait(1000)
-                    end
-                    local data = originalFun()
-                    local job = data.job
-                    local gang = data.gang
-                    local charinfo = data.charinfo
+function Core.getPlayerData()
+    local player = shared.GetPlayerData()
```

Comment:
Please test yourself. Dont think this will work

Reply:
isnt it this the way to pull the playerdata?